Resources/Prototypes/Entities/Objects/Shields/shields.yml

```diff
@@ -217,3 +217,91 @@
                 SheetGlass:
                   min: 5
                   max: 5
+
+- type: entity
+  name: energy shield
+  parent: BaseItem
+  id: EnergyShield
+  description: Exotic energy shield that reflects almost all laser beams. But only them.
```

Comment:
maybe make it differ from the uplink's description so people who are not supposed to know what it does don't actually know it via description?

Reply:
ok